tests/test_api.py

```diff
@@ -197,6 +199,38 @@ def test_do_GET(self):
+        #
+        with patch.object(RestApiHandler, 'get_postgresql_status', Mock(return_value={'role': 'master'})):
+            MockRestApiServer(RestApiHandler, 'GET /master?lag=1M&tag_key1=true&tag_key2=false&tag_key3=1&tag_key4=1.4&tag_key5=RandomTag', patroni=mock_patroni)
+            MockRestApiServer(RestApiHandler, 'GET /master?lag=1M&tag_key1=true&tag_key2=False&tag_key3=1&tag_key4=1.4&tag_key5=RandomTag', patroni=mock_patroni)
+            MockRestApiServer(RestApiHandler, 'GET /master?lag=1M&tag_key1=true&tag_key2=false&tag_key3=1.0&tag_key4=1.4&tag_key5=RandomTag', patroni=mock_patroni)
+            MockRestApiServer(RestApiHandler, 'GET /master?lag=1M&tag_key1=true&tag_key2=false&tag_key3=1&tag_key4=1.4&tag_key5=RandomTag&tag_key6=RandomTag2', patroni=mock_patroni)
+        #
+        MockRestApiServer(RestApiHandler, 'GET /replica?lag=1M&tag_key1=true&tag_key2=false&tag_key3=1&tag_key4=1.4&tag_key5=RandomTag', patroni=mock_patroni)
+        MockRestApiServer(RestApiHandler, 'GET /replica?lag=1M&tag_key1=true&tag_key2=False&tag_key3=1&tag_key4=1.4&tag_key5=RandomTag', patroni=mock_patroni)
+        MockRestApiServer(RestApiHandler, 'GET /replica?lag=1M&tag_key1=true&tag_key2=false&tag_key3=1.0&tag_key4=1.4&tag_key5=RandomTag', patroni=mock_patroni)
+        MockRestApiServer(RestApiHandler, 'GET /replica?lag=1M&tag_key1=true&tag_key2=false&tag_key3=1&tag_key4=1.4&tag_key5=RandomTag&tag_key6=RandomTag2', patroni=mock_patroni)
+        #
+        with patch.object(RestApiHandler, 'get_postgresql_status', Mock(return_value={'role': 'master'})):
+            MockRestApiServer(RestApiHandler, 'GET /replica?lag=1M&tag_key1=true&tag_key2=false&tag_key3=1&tag_key4=1.4&tag_key5=RandomTag', patroni=mock_patroni)
+            MockRestApiServer(RestApiHandler, 'GET /replica?lag=1M&tag_key1=true&tag_key2=False&tag_key3=1&tag_key4=1.4&tag_key5=RandomTag', patroni=mock_patroni)
+            MockRestApiServer(RestApiHandler, 'GET /replica?lag=1M&tag_key1=true&tag_key2=false&tag_key3=1.0&tag_key4=1.4&tag_key5=RandomTag', patroni=mock_patroni)
+            MockRestApiServer(RestApiHandler, 'GET /replica?lag=1M&tag_key1=true&tag_key2=false&tag_key3=1&tag_key4=1.4&tag_key5=RandomTag&tag_key6=RandomTag2', patroni=mock_patroni)
+        #
+        MockRestApiServer(RestApiHandler, 'GET /read-write?lag=1M&tag_key1=true&tag_key2=false&tag_key3=1&tag_key4=1.4&tag_key5=RandomTag', patroni=mock_patroni)
+        MockRestApiServer(RestApiHandler, 'GET /read-write?lag=1M&tag_key1=true&tag_key2=False&tag_key3=1&tag_key4=1.4&tag_key5=RandomTag', patroni=mock_patroni)
+        MockRestApiServer(RestApiHandler, 'GET /read-write?lag=1M&tag_key1=true&tag_key2=false&tag_key3=1.0&tag_key4=1.4&tag_key5=RandomTag', patroni=mock_patroni)
+        MockRestApiServer(RestApiHandler, 'GET /read-write?lag=1M&tag_key1=true&tag_key2=false&tag_key3=1&tag_key4=1.4&tag_key5=RandomTag&tag_key6=RandomTag2', patroni=mock_patroni)
```

Comment:
Lines are clearly bigger than 120 characters, It will not pass flake8 validation (you can call it as `python setup.py flake8`)